bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -479,6 +480,24 @@ static function (Task $task) {
             $objectData['general']['versionDate'] = $objectFromDatabase->getModificationDate();
             $objectData['general']['versionCount'] = $objectFromDatabase->getVersionCount();
 
+            $userOwner = User::getById($objectData['general']['o_userOwner']);
+            if (empty($userOwner)) {
```

Comment:
I think we should make this a bit more generic, what about adding a `getFullName()` method in the `User` object and maybe a trait that handles all the displaying related stuff?